projects/kit/pipes/stringify/stringify.pipe.spec.ts

```diff
@@ -0,0 +1,9 @@
+import {TuiStringifyPipe} from './stringify.pipe';
+
+describe('TuiStringify pipe', () => {
+    it('works', () => {
+        const pipe = new TuiStringifyPipe();
+
+        expect(pipe.transform('data')({data: 237, value: 327})).toEqual('237');
```

Comment:
What about pass with `null`, `undefined` or `empty string` to transform? ))

Reply:
Added a couple more tests

Reply:
what about test with invalid key)) not `data`

Reply:
TypeScript would throw

Reply:
![image](https://user-images.githubusercontent.com/11832552/134881374-d552d72f-382b-4cd1-bdfc-70d3006ee74f.png)

Reply:
oh, I see)